src/gui/win.rs

```diff
@@ -391,6 +749,50 @@ impl SystemTray {
             error!("no CFG var that contains active kanata config");
         };
     }
+    /// Check if tooltip data is changed, and update tooltip window size / timer duration
+    fn update_tooltip_data(&self, k: &MutexGuard<Kanata>) -> bool {
```

Comment:
Is there a need for this to be a `&MutexGuard` instead of a normal `&` reference?